### PR TITLE
doc: improve plugin, webui, virtualfull chapters

### PR DESCRIPTION
**Pull request by bruno-at-bareos**
This PR is a composite PR for several documentation changes. It is proposed to also backport this to branch-22

This PR propose to change `Plugin Names = "python"` samples to `Plugin Names = "python3"` in documentation: making it clear to everyone that parameter need to be adjusted depending on the python version you use.

This PR contain also the additional how-to enable php-fpm for apache in debian like distributions.

This PR contain the documentation changes about Virtual Full Backup can only be run on the same bareos-sd daemon.

This PR contain the documentation changes for bareos-webui installation under SLE12SP5 adding the PackageHub repository.

OP#5425
OP#5426
OP#5433
OP#5431

#### Please check

- [X] Short description and the purpose of this PR is present _above this paragraph_
- [X] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

### Checklist for the _reviewer_ of the PR (will be processed by the Bareos team)
Make sure you check/merge the PR using `devtools/pr-tool` to have some simple automated checks run and a proper changelog record added.

##### General
- [x] Is the PR title usable as CHANGELOG entry?
- [x] Purpose of the PR is understood
- [x] Commit descriptions are understandable and well formatted
- [x] Check backport line

##### Source code quality
- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR

### CODE REVIEW

File: docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst [hunk 1]
```diff
@@ -106,6 +106,18 @@ After adding the repository simply install the bareos-webui package via your pac
 
       apt-get install bareos-webui
 
+Because php-fpm support is not automatically added to Apache2 on Debian like platform, you have to issue those commands to enable it.
+Replace :command:`php8.1-fpm` by the version you have installed.
+
+.. code-block:: shell-session
+   :caption: Debin,Ubuntu enabling php8-fpm support on Apache2 example
```
Comment (sduehr) [inline docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst:113]:
```suggestion
   :caption: Debian, Ubuntu enabling php8-fpm support on Apache2 example
```

**Comment by bruno-at-bareos** [comment]
> Thanks, only few changes requested, and could you please also change to Plugin Names = python3 in VMwarePlugin.rst.inc?

Done in last push (forced then) added to first commit.

**Review by sduehr — APPROVED** [review]
Thanks, looks good now.

**Comment by bruno-at-bareos** [comment]
We just add the fact that we want to add also the PackageHub needs for SL12SP5

File: docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst [hunk 2]
```diff
@@ -65,7 +65,7 @@ System Requirements
 
 -  PHP 7 or newer is recommended.
 
--  On SUSE Linux Enterprise 12 you need the additional SUSE Linux Enterprise Module for Web Scripting 12.
+-  On SUSE Linux Enterprise 12 you need the additional SUSE Linux Enterprise Module for Web Scripting 12 and additionnal repository :strong:`Package Hub` to satisfy apache2-mod-fcgid requirement.
```
Comment (sduehr) [inline docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst:68]:
```suggestion
-  On SUSE Linux Enterprise 12 you need the additional SUSE Linux Enterprise Module for Web Scripting 12 and additional repository :strong:`Package Hub` to satisfy apache2-mod-fcgid requirement.
```

**Review by sduehr — CHANGES_REQUESTED** [review]
Thanks, only found a typo.

File: docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst (hunk 2 above)
Comment (bruno-at-bareos) [reply docs/manuals/source/IntroductionAndTutorial/BareosWebui.rst]: Fixed. sorry.

**Review by sduehr — APPROVED** [review]
Thanks, looks good.

**Pull request merged**